cpp/include/cudf/column/column_view.hpp

```diff
@@ -527,4 +529,17 @@ class mutable_column_view : public detail::column_view_base {
  **/
 size_type count_descendants(column_view parent);
 
+/**
+ * @brief Zero-copy cast between binary compatible types
+ *
+ * Can be used to cast timestamp columns to their underlying integer representation
```

Comment:
Should you mention the decimal fixed-point use case also?

```suggestion
 * Can be used to cast timestamp columns to their underlying integer representation.
```

Reply:
Yes, but I was planning on adding that in a follow-up PR when support for `fixed_point` is available.